tests/plugins/merges.test.ts

```diff
@@ -332,7 +332,7 @@ describe("merges", () => {
       }
     }
     const fixture = makeTestFixture();
-    const app = new App(Parent);
```

Comment:
This tests the interactiveMerge, which has nothing to do with the plugins. I'm pretty sure it's already tested in the file helpers/ui.test.ts ...
If its the case, can you remove the test completely ?